frontend/src/selectors/collectives.js

```diff
@@ -94,7 +107,7 @@ export const getPopulatedCollectiveSelector = createSelector(
     getCollectiveHostSelector,
     getCollectiveMembersSelector,
     getCollectiveBackersSelector,
-    getCollectiveContributorsSelector,
+    getCollectiveGithubContributorsSelector,
     getUnpaidExpensesSelector,
     getPaidExpensesSelector,
     getTransactionsSelector ],
```

Comment:
Instead of accessing subCollectiveSelector directly, you could also put it into 'getPopulatedCollectiveSelector' and then any page could tell if it's a super collective or not.